devito/core/gpu_openmp.py

```diff
@@ -84,7 +93,10 @@ def _map_release(cls, f):
     @classmethod
     def _map_delete(cls, f):
         return cls.lang['map-exit-delete'](f.name, ''.join('[0:%s]' % i
-                                                           for i in cls._map_data(f)))
+                                                           for i in cls._map_data(f)),
```

Comment:
this, written this way over 5 lines, is a bit chaotic. Can you rewrite it more cleanly?

Reply:
I managed to reduce it to 4 lines.